test/e2e/framework/node/resource.go

```diff
@@ -48,6 +52,14 @@ const (
 	proxyTimeout = 2 * time.Minute
 )
 
+// podNode is a pod-node pair indicating which node a given pod is running on
+type podNode struct {
+	// Pod represents pod name
+	pod string
+	// Node represents node name
+	node string
+}
```

Comment:
Unexported because no external package reference.

Reply:
revert this to exported as per golint warning: `exported func PodNodePairs returns unexported type []node.podNode, which can be annoying to use`